cluster/cluster.go

```diff
@@ -17,6 +17,12 @@ var (
 
 	errClusterInitialized    = errors.New("openstorage.cluster: already initialized")
 	errClusterNotInitialized = errors.New("openstorage.cluster: not initialized")
+
+	// Inst returns an instance of an already instantiated cluster manager.
+	// This function can be overritten for testing purposes
```

Comment:
Why not just add Set and Get for the global "inst" object? It is more intuitive than setting the Inst function object.

Reply:
Since it is just for testing and it is a global variable, I didn't see a need.